README.md

```diff
@@ -1,4 +1,4 @@
-# WordPress Playground and  PHP WASM (WebAssembly)
+# WordPress Playground and PHP WASM (WebAssembly)
```

Comment:
This one is my bad, a rebase should  fix it

Reply:
Oops, I've been trying to keep unrelated changes unstaged/uncommitted to this PR, but it must have slipped through. Oh, I see it happened when I merged from upstream trunk.